CONTRIBUTING.md

```diff
@@ -255,14 +255,20 @@ For more, please read the article [How to Run GitHub Actions Locally with the ac
 
 ## Publishing
 
-This project uses GitHub Actions to publish the packages automatically to npm. New packages are published after the new tag is pushed to the main branch. PR can be merged only by the appropriate group of maintainers.
+This project uses GitHub Actions to publish the packages automatically to npm.
+New packages are published after the new tag is pushed to the main branch.
+PR can be merged only by the appropriate group of maintainers.
 
 ### Steps to Create a New Package Version
 
-1. Merge all appropriate PRs you want to publish into the main branch
+1. Merge all appropriate PRs you want to publish into the appropriate branch
+   - branches:
+     - `main` - for the latest stable version
```

Comment:
what is the plan with this list? Do we know if we will add something here?

Reply:
Yeah, I hope so. We have more discussions about release management ahead.